### PR TITLE
added count_positive_and_negative.py

### PR DESCRIPTION
**Pull request by kapilkumar2001**
count positive and negative elements in the python list

### CODE REVIEW

File: Basic-Scripts/count_positive_and_negative.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Python program to count positive and negative numbers in a List 
+  
+# list of numbers 
+list1 = [10, -21, 4, -45, 66, -93, 1]
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py]: Take input from user

File: Basic-Scripts/count_positive_and_negative.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  
+# iterating each number in list 
+for num in list1:     
+    # checking condition 
+    if num >= 0: 
+        positive_count += 1
+    else: 
+        negative_count += 1
+          
+print("Positive numbers in the list: ", positive_count)
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py]: Include __name__ == "___main__"

Enclose your algo inside a function and call it directly from main

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/count_positive_and_negative.py (hunk 2 above)
Comment (kapilkumar2001) [reply Basic-Scripts/count_positive_and_negative.py]: done

File: Basic-Scripts/count_positive_and_negative.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    for num in list1:                           # iterating each number in list 
+        if num>=0:                            # checking condition 
+            pos_count += 1
+        else: 
+            neg_count += 1      
+    print("Positive numbers in the list: ", pos_count) 
+    print("Negative numbers in the list: ", neg_count) 
+
+#main program
+n= int(input("Enter number of elements: "))
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py]:
```suggestion
if __name__ == "__main__":
    num = int(input("Enter number of elements: "))
```

File: Basic-Scripts/count_positive_and_negative.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+#list of numbers
+lst = []
+for i in range(0,n):
+    #print() 
+    ele=int(input("Enter "+str(i)+"th element: "))
+    lst.append(ele)
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py]:
```suggestion
for index in range(0,num):
    #print() 
    ele=int(input("Enter "+str(index)+"th element: "))
    lst.append(ele)        
```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/count_positive_and_negative.py (hunk 4 above)
Comment (kapilkumar2001) [reply Basic-Scripts/count_positive_and_negative.py]: okay done

File: Basic-Scripts/count_positive_and_negative.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Python program to count positive and negative numbers in a List given by user 
+def count_pos_neg(list1):
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py:2]: Include Docstring Documentation of your function.

File: Basic-Scripts/count_positive_and_negative.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+#main program
+if __name__ == "__main__":
+    num = int(input("Enter number of elements: "))
+#list of numbers
+lst = []
+for index in range(0,num):
+    #print() 
+    ele=int(input("Enter "+str(index)+"th element: "))
+    lst.append(ele)        
+count_pos_neg(lst)
```
Comment (SKAUL05) [inline Basic-Scripts/count_positive_and_negative.py]:
```suggestion
    lst = []
    for index in range(0,num):
        #print() 
        ele=int(input("Enter "+str(index)+"th element: "))
        lst.append(ele)        
    count_pos_neg(lst)
```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/count_positive_and_negative.py (hunk 6 above)
Comment (kapilkumar2001) [reply Basic-Scripts/count_positive_and_negative.py]: done

Comment (kapilkumar2001) [reply Basic-Scripts/count_positive_and_negative.py]: please, check it now sir

**Review by SKAUL05 — APPROVED** [review]
Good to go @kapilkumar2001 
Review & Merge @ankitdobhal

**Pull request merged**